report/sections/results.tex

```diff
@@ -1 +1,37 @@
 \section{Results}
+\section{Discretization of PID controller}
```

Comment:
Shouldn't this be in the implementation section or somewhere similar? I figure we would talk about the results of the project in this section.

Reply:
I can push all of this in the implementation section and solving the issues